hack/build/docker/builder/Dockerfile

```diff
@@ -24,7 +24,17 @@ RUN \
     ( go get -d github.com/mattn/goveralls && \
     cd $GOPATH/src/github.com/mattn/goveralls && \
     go install ./... ) && \
-    ( go get -u golang.org/x/lint/golint )
+    ( go get -u golang.org/x/lint/golint ) && \
+    ( go get -u github.com/mhenriks/go-swagger-utils/swagger-doc ) && \
```

Comment:
Sure we want your private repo in here?

Reply:
It's either mine or @rmohr's!  Should prob use Roman's though as that's what KubeVirt uses.

Reply:
If you can, go with the official openapi-gen from k8s. We had an issue blocking us in the past to switch. Since my code generator also just works for us right now we did not switch yet. Here the PR: https://github.com/kubevirt/kubevirt/pull/1419.

Reply:
Oh, it might only generate swagger v3. Forgot that. Not sure if you need v2 or v3.